Client/src/Pages/Auth/Login.jsx

```diff
@@ -168,26 +75,8 @@ const StepOne = ({ form, errors, onSubmit, onChange, onBack }) => {
 					/>
 					<Stack
 						direction="row"
-						justifyContent="space-between"
+						justifyContent="flex-end"
 					>
-						<Button
-							variant="outlined"
-							color="info"
-							onClick={onBack}
```

Comment:
`onBack` is no longer used, it can be removed from `StepOne` props

Reply:
Fixed in: https://github.com/bluewave-labs/checkmate/commit/ba8cc04f2d58cfc0320be926d4af5776a70f7b80